cumulusci/cli/cci.py

```diff
@@ -961,6 +977,14 @@ def org_list(config, plain):
                 "username", org_config.userinfo__preferred_username
             )
             row.append(username)
+            if org_config.expires:
+                if org_config.expires != "Persistent":
+                    row.append(org_config.expires)
+                else:
+                    row.append(org_config.expires)
+            else:
+                row.append("Unknown")
```

Comment:
What's going on here? You're doing the same thing in the `if` and `else` blocks of the `if org_config.expires != "Persistent":` condition. This entire thing can be simplified down to:

    row.append(org_config.expires or "Unknown")